packages/component-base/src/vaadin-a11y-announcer.js

```diff
@@ -0,0 +1,158 @@
+/**
```

Comment:
Is the controller + Web Component combination an overkill for what we need? Would it be enough to have `vaadin-a11y-announcer.js` module that creates a single `<div style="...">` under the body and then updates it on demand. A client, such as the vaadin-date-picker, could then use the announcer with:

```js
import { announce } from '@vaadin/component-base/src/vaadin-a11y-announcer.js'

announce('some announcement');

or

announce('some announcement', { mode: 'polite', timeout: 1000});
```

Reply:
Yes, the web component isn't really needed. The main question is when to create the `<div>` and the controller kinda solves that by using `hostConnected` so maybe we could keep it.

I will update the PR to get rid of the component.

Reply:
> The main question is when to create the `<div>`

Couldn't that be created inside the module? Similarly to https://github.com/vaadin/web-components/blob/master/packages/component-base/src/resize-mixin.js#L8

Reply:
Done.